etc/kayobe/ansible/drac-firmware-update.yml

```diff
@@ -0,0 +1,64 @@
+---
+- hosts: overcloud
+  gather_facts: false
+  connection: local
+  serial: "{{ lookup('env', 'ANSIBLE_SERIAL') | default(1, true) }}"
+
+  collections:
+   - dellemc.openmanage
+
+  vars:
+    # Run with ``-e dell_drm_apply_update=true`` to apply the firmware updates.
+    dell_drm_apply_update: False
+    dell_drm_address: "https://{{ oob_oc_net_name | net_ip(inventory_hostname=groups['seed'][0]) }}"
```

Comment:
The seed is assumed here when previously the criteria was "Choose a host where it won't conflict with another service". I think you should either add a dell_drm group or make it clear that the seed is the default host.